01/app.js

```diff
@@ -0,0 +1,10 @@
+const user = {
+  firstName: 'Sebastian',
+  lastName: 'Matysiak',
+  sex: 'Male',
+  age: '23',
+};
```

Comment:
👍

Reply:
Potwierdzam, lepiej będzie jeśli będzie `numer`.
Jednak w tym zadaniu nie było to istotne :)